s3/src/main/scala/akka/stream/alpakka/s3/javadsl/S3Client.scala

```diff
@@ -56,9 +56,14 @@ final class S3Client(s3Settings: S3Settings, system: ActorSystem, mat: Materiali
     impl.download(S3Location(bucket, key), Some(scalaRange)).asJava
   }
 
-  // #list-bucket
```

Comment:
Paradox still can not find the snippet because it was `#list-bucket` and now it is `#listBucket` in the `S3SourceSpec.scala`. After changing to `#list-bucket` this should be good to go in.